test/helpers.ts

```diff
@@ -0,0 +1,45 @@
+import Web3 from 'web3';
+import { DydxClient, EthereumAccount } from '@dydxprotocol/v3-client';
+import { generateKeyPairUnsafe } from '@dydxprotocol/starkex-lib';
+import axios, { AxiosRequestConfig } from 'axios';
+
+export async function createUser() {
+  const web3 = new Web3();
+  const account: EthereumAccount = web3.eth.accounts.wallet.create(1)[0];
+  const address: string = account.address.toLowerCase();
```

Comment:
You don't need to create a new ethereum account with web3, can you use `getSigner` instead (given you're already using `hardhat-ethers`)? https://github.com/nomiclabs/hardhat/tree/master/packages/hardhat-ethers